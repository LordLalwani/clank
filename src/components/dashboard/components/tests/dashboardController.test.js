import { cleanup } from "@testing-library/react";
import React from "react";
import renderer from "react-test-renderer";
import DashboardController from "../dashboardController";

afterEach(cleanup);

describe("render tests", () => {
  it("should render DashboardController component correctly", () => {
    const tree = renderer.create(<DashboardController />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
