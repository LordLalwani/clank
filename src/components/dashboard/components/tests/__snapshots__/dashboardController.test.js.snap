// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render tests should render DashboardController component correctly 1`] = `
<div
  style={
    Object {
      "width": "100vw",
    }
  }
>
  <main
    className="Dashboard-content-2"
  >
    <div
      className="Dashboard-toolbar-1"
    />
    <p>
      dashboard
    </p>
  </main>
</div>
`;
